api/swagger-spec/swagger.yml

```diff
@@ -275,6 +301,13 @@ definitions:
         type: array
         items:
           $ref: "#/definitions/componentVersion"
+  doctorInfo:
+    type: object
+    required:
+      - cluster
+    properties:
+      cluster:
+        $ref: "#/definitions/cluster"
```

Comment:
>{
    id: {uuid}
    cluster_id: {cluster id}
    components: {componentVersion}
    data: {generic object}
}


That's basically splitting cluster data structure